src/cf/app.go

```diff
@@ -0,0 +1,93 @@
+package cf
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"code.cloudfoundry.org/lager"
+
+	"models"
+)
+
+const (
+	TokenTypeBearer = "bearer"
+	PathAppSummary  = "/v2/apps/{appid}/summary"
+	PathAppUpdate   = "/v2/apps/{appid}"
+)
+
+func (c *cfClient) GetAppInstances(appId string) (int, error) {
+	url := c.conf.Api + strings.Replace(PathAppSummary, "{appid}", appId, 1)
```

Comment:
Let us use the PathAppUpdate instead. Summary is more expensive and unnecessary to get the instance count.